tide_grant.module

```diff
@@ -145,3 +146,32 @@ function tide_grant_migration_plugins_alter(array &$migrations) {
     $migrations['grants_feed_importer']['source']['urls'] = $url;
   }
 }
+
+/**
+ * Implements hook_node_insert().
+ */
+function tide_grant_node_insert(NodeInterface $node) {
+  if ($node->bundle() != 'grant') {
+    return;
+  }
+  // Everytime the feed will import contents, this will trigger.
+  // Only update the grant nodes created by anonymous user.
+  if ($node->getOwnerId() == 0) {
```

Comment:
Hey @MdNadimHossain I think this is really well done. The only comment I'd make (for future changes, don't need to do it for this) is that you're writing the exact same code in the update hook which probably makes it a candidate for an extract to a function.

Reply:
@anthony-malkoun ok you mean writing it as a function and then call it once in the update hook.